panther-labs/panther
# Sort by displayName by default instead of ID

**Description in progress**

## Background
closes: https://github.com/panther-labs/panther-enterprise/issues/434

## Changes

- Add backend sort by `displayName` functionality
- Add frontend option to sort by `displayName` in dropdown of rules page
- Add tests

## Testing

- Load rules page to ensure name sort is default
- Select previously implemented sort, then select name sort
- Select name sort when list is already sorted by name
